Write metatext (#189)
SW709E19/Report: report/PartProblemDomain/PartCtrl.tex

> This chapter documents the application domain and a use case diagram with its corresonding workflow description. 

Reviewer: ```suggestion
This chapter documents user stories and actors in the application domain and presents a use case diagram with a corresonding workflow description. 
```

> This chapter presents the problem domain. The first \autoref{cha:user-stories-and-actors} introduces what user stories are and then goes through the different actors that have an interest in the system.

Reviewer 2: ```suggestion
This chapter documents the application domain. The first \autoref{cha:user-stories-and-actors} introduces user stories and actors.
```

> The use case diagram and workflow in \autoref{cha:usecase-diagram-chapter} then goes through the relation between the actors and the possible actions.

Reviewer 2: ```suggestion
The use case diagram and workflow is documented in \autoref{cha:usecase-diagram-chapter}, which goes through the relation between the actors and their responsibilities.
```
